hazelcast/src/main/java/com/hazelcast/config/XmlConfigBuilder.java

```diff
@@ -1726,6 +1726,21 @@ private RingbufferStoreConfig createRingbufferStoreConfig(Node node) {
         return config;
     }
 
+    private MergePolicyConfig createMergePolicyConfig(Node node) {
+        MergePolicyConfig mergePolicyConfig = new MergePolicyConfig();
+        String policyString = getTextContent(node).trim();
+        mergePolicyConfig.setPolicy(policyString);
+        NamedNodeMap attributes = node.getAttributes();
```

Comment:
Can you replace this with something like this:
```
        final String att = getAttribute(node, "batch-size");
        if (att != null) {
            mergePolicyConfig.setBatchSize(getIntegerValue("batch-size", att));
        }
```

Reply:
We might have to implement more attributes, e.g. the `discardNewDistributedObjects` or a switch to skip the search for existing entries (optimization for key-less data structures like queue, list, set, where it's very expensive to find the existing entry).

But yeah, we could change that code again, if we have to. I will apply your optimization for now.